firefox-ios/Client/Frontend/Browser/TabScrollController.swift

```diff
@@ -441,14 +441,14 @@ private extension TabScrollController {
     /// - `.visible`: Toolbars are currently showing (`toolbarsShowing == true`).
     /// - `.animating`: In transition or partially visible state.
     func updateToolbarState() {
-        // bottom containers are collapsedd
+        // bottom containers are collapsed
```

Comment:
nit: Can we improve this comment with something like `// Checks if bottom containers are fully collapsed based on their offsets`